plugin-server/tests/cdp/fixtures.ts

```diff
@@ -23,6 +23,18 @@ export const createHogFunction = (hogFunction: Partial<HogFunctionType>) => {
     return item
 }
 
+export const createIntegration = (integration: Partial<IntegrationType>) => {
```

Comment:
```suggestion
export const createIntegration = (integration: Partial<IntegrationType>, team_id: number) => {
```

🤷

Reply:
Not sure that makes sense in this case (this is just for creating test fixtures)